docs/mockwbemserver.rst

```diff
@@ -199,31 +200,32 @@ The mock WBEM server does NOT support:
 
 Diagram of flow of requests operations through mocker
 
-.. _`Faked WBEM operations`:
 
-Faked WBEM operations
----------------------
+.. _`WBEM operations of a mock WBEM server`:
+
+WBEM operations of a mock WBEM server
+-------------------------------------
 
-The :class:`pywbem_mock.FakedWBEMConnection`  class supports the same WBEM
+The :class:`pywbem_mock.FakedWBEMConnection` class supports the same WBEM
 operations that are supported by the :class:`pywbem.WBEMConnection` class and
-in addition a set of methods to execute WBEM server responders for each of these
-client methods using an internal CIM repository of CIM classes, CIM instances,
-and CIM qualifier declarations.
+in addition a set of methods for setting up its mocked CIM repository, and
+for registering user-defined WBEM operation responders ("providers") for
+some the WBEM operations.
 
-These faked operations generally adhere to the behavior requirements defined in
-:term:`DSP0200` for handling input parameters and returning a result.
+These *faked WBEM operations* generally adhere to the behavior requirements
+defined in :term:`DSP0200` for handling input parameters and returning a result.
 
-The faked operations get the data to be returned from the mock repository of
-the faked connection, and put the data provided in operation parameters that
+The faked WBEM operations get the data to be returned from the mock repository of
```

Comment:
I think we are mixing the wording repository and phrase cim repostiory in different parts of our documentation.  Thus, here it could be mock CIM repository.  I would not get carried away trying for absolute consistency since the meaning is clear

Reply:
I agree that we do, and suggest the following terms:

* "CIM repository of the mock WBEM server" - fully specific term used when that context is needed.
* "mock repository" - short term for the above, that we define in the glossary that way, and introduce it on first usage in each main chapter.

Reply:
We agreed to consolidate on the following terms:
* "CIM repository of the mock WBEM server" - fully specific term used when that context is needed.
* "mock repository" - short term for the above, that we define in the glossary that way, and introduce it on first usage in each main chapter.
* "provider" (subsumes "responder"), define in glossary, also relative to SNIA providers
* "provider method"  (subsumes "responder method" and "Python operation method")

Reply:
I have some second thoughts on the term "mock provider": It is currently used much less than "CIM repository", and "CIM repository" seems to better hit the nail, IMO. So I will use "CIM repository", instead.

Reply:
Karl agrees